integrations/db-mongodb/src/MongoDb.ts

```diff
@@ -0,0 +1,103 @@
+import { Collection, Db, Document, MongoClient } from 'mongodb';
+import {
+  DbItem,
+  DbPlugin,
+  DbPluginConfig,
+  HandleRequest,
+  Jovo,
+  PersistableSessionData,
+  PersistableUserData,
+  RequiredOnlyWhere,
+} from '@jovotech/framework';
+
+export interface MongoDbConfig extends DbPluginConfig {
+  /** Specify username, password and clusterUrl. Additional parameters can also be added. See https://docs.mongodb.com/drivers/node/current/fundamentals/connection/#connection-uri for more details */
+  connectionString: string;
+  /** The name of the database we want to use. If not provided, use database name from connection string. A new database is created if doesn't exist yet. */
+  databaseName?: string;
+  /** A new collection is created with that name if doesn't exist yet. */
+  collectionName?: string;
+}
+
+export type MongoDbInitConfig = RequiredOnlyWhere<MongoDbConfig, 'connectionString'>;
+
+export interface MongoDbItem {
+  id: string;
+  user?: PersistableUserData;
+  session?: PersistableSessionData;
+  createdAt?: string;
+  updatedAt?: string;
+}
+
+/** Default database name in MongoDB. if no name was specified, 'test' is used. See https://docs.mongodb.com/manual/tutorial/getting-started/#getting-started */
+export const MONGODB_DEFAULT_DATABASE_NAME = 'test';
+export const JOVO_DEFAULT_DATABASE_NAME = 'jovo_db';
+export const JOVO_DEFAULT_COLLECTION_NAME = 'jovoUsers';
```

Comment:
Is there a specific reason why `JOVO_DEFAULT_DATABASE_NAME` uses `jovo_db` (snake_case) and  `JOVO_DEFAULT_COLLECTION_NAME` uses `jovoUsers` (camelCase)? Does this have something to do with MongoDB naming conventions?

Reply:
Because the examples in MongoDb Atlas were using snake case for DB and camel case for collections. 
Seems like camel case is the convention.